### DISCUSSION TITLE
Alert through Proxy or customize request

### DISCUSSION THREAD

**Discussion by Thaiph1308** [openobserve/openobserve#2042] [Ideas]
Hello Openobserve team,
I'm using Openobserve to centralize and monitor logs, but in my case, it requires traffic to go through a proxy before reaching Teams webhook. It would be perfect if Openobserve could have a feature like this (optional Proxy or let the user customize the request).

**Comment by prabhatsharma** [comment]
https://github.com/openobserve/openobserve/issues/2043
